docs/source/workflows/descriptors.rst

```diff
@@ -62,9 +62,18 @@ The Molecular Structure Descriptor has no parameters other than a name.
 Formulation Descriptor
 ------------------------------
 
-:class:`~citrine.informatics.descriptors.FormulationDescriptor` is used to represent variables that contain information
-about mixtures of other materials.
-The Formulation Descriptor has no parameters other than a name.
+:class:`~citrine.informatics.descriptors.FormulationDescriptor`
+is used to represent variables that contain information about mixtures of other materials.
+The Formulation Descriptor has no parameters other than a name,
+of which the two allowed values are 'Formulation' and 'Flat Formulation'.
+The key 'Formulation' should be used when referring to mixtures found directly in the training data,
+for which the ingredients may be other mixtures themselves.
+The key 'Flat Formulation' should be used when referring to mixtures containing only raw ingredients,
```

Comment:
In a one-level mixing problem the mixture only contains raw ingredients, but we still want to use "Flat Formulation"